### PR TITLE
Copyediting for distributed transactions

### PR DESCRIPTION
**Pull request by Hoverbear**
As discussed in https://github.com/tikv/deep-dive-tikv/pull/12#issuecomment-445024456.

### CODE REVIEW

**Comment by brson** [comment]
Thanks for posting this @Hoverbear

**Comment by brson** [comment]
PTAL @zhangjinpeng1987

File: src/distributed-transaction/introduction.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Introduction
 
-TiKV is a distributed transactional key-value database, transaction is a foundational function for TiKV. This chapter we will talk about general implementations about distributed transaction and some implementation details in TiKV.
+As TiKV is a distributed transactional key-value database, transactions are a core feature of TiKV. In this chapter we will talk about general implementations of distributed transaction and some implementation details in TiKV.
```
Comment (zhangjinpeng87) [inline src/distributed-transaction/introduction.md]: s/transactions are a core feature of TiKV/transaction is a core feature of TiKV

**Comment by zhangjinpeng87** [comment]
Other parts LGTM.

**Comment by brson** [comment]
Updated per @zhangjinpeng1987.

**Pull request merged**